### ISSUE TITLE
URL invalid when importing whole .txt file

### ISSUE THREAD

**Issue by shensmobile** [heartexlabs/label-studio#1301]
**Describe the bug**
When I set valueType=url and go to import a .txt file for NER labelling (as a time series), I get the following error when I go to open the task:

> Error
> Do not put text directly in task data if you use valueType=url.
> URL (/data/upload/15203235_RxI9qSk.txt) is not valid.

If I go to localhost:8080/data/upload/15203235_RxI9qSk.txt, I do get the text file in its entirety.  

Is there anything special I have to do when importing the text file?  Would it be easier to just put all the .txt files into a .csv and upload them that way?

**Comment by smoreface** [comment]
@shensmobile it would work to put the text in CSV. Can you share your labeling config? Did you import the text file as a time series or list of tasks?

**Comment by makseq** [comment]
@shensmobile Just checked it with the latest LS version (1.2 from the master branch) and this config: 

```
<View>
  <Text name="text" value="$text" valueType="url"/>
  <View style="box-shadow: 2px 2px 5px #999; padding: 20px; margin-top: 2em; border-radius: 5px;">
    <Header value="Choose text sentiment"/>
    <Choices name="sentiment" toName="text" choice="single" showInLine="true">
      <Choice value="Positive"/>
      <Choice value="Negative"/>
      <Choice value="Neutral"/>
    </Choices>
  </View>
</View>
```

and it works fine. 

![image](https://user-images.githubusercontent.com/501780/133341548-44fbfdf7-aca1-430e-bde1-e7f370874e03.png)

**Comment by SapienMachineLearning** [comment]
@smoreface I ended up putting all the text into a .csv as a workaround.  I had been importing the text files as time series though.  I don't have the labelling config I was using at the time, sorry :(

@makseq I'm on version 1.1.0, I tried that labelling config, and still I get an error (when my text is imported as a time series).

```
Error
Do not put text directly in task data if you use valueType=url.

URL (/data/upload/15203235_G5czSTB.txt) is not valid.```

**Comment by shensmobile** [comment]
@smoreface I did end up putting all the .txt into a CSV to make it work.  I was just curious why text was not working in my case, more of curiosity than anything.  I don't have the original labelling config I used, sorry, but I was importing it as a times series, not a list of tasks.

@makseq I am unfortunately on version 1.1.0.  I tried that labelling config anyway and get this error:

```
Error
Do not put text directly in task data if you use valueType=url.

URL (/data/upload/15203235_G5czSTB.txt) is not valid.```

I should update to 1.2.0 and try again.  Thank you!
